# HG changeset patch
# User Ray Donnelly <mingw.android@gmail.com>
# Date 1471184922 -3600
#      Sun Aug 14 15:28:42 2016 +0100
# Node ID 8bceceff4152995846d61539c238f5f433c1d723
# Parent  dd84aef4c2166c6fbf63fd75dc567d572bda3fae
Win32: Add missing jump_to_toplevel() extern

diff --git a/rpy/rinterface/_rinterface.c b/rpy/rinterface/_rinterface.c
--- a/rpy/rinterface/_rinterface.c
+++ b/rpy/rinterface/_rinterface.c
@@ -62,6 +62,9 @@
 
 #if !(defined(Win32) || defined(Win64))
 #include <Rinterface.h>
+#else
+#define jump_to_toplevel Rf_jump_to_toplevel
+extern __attribute__ ((dllexport)) void jump_to_toplevel();
 #endif
 #include <R_ext/Complex.h>
 #include <Rembedded.h>
